src/Console/Commands/CashierUpdate.php

```diff
@@ -36,7 +42,38 @@ public function handle()
 
         $this->comment('Publishing Cashier v2 migrations...');
         $this->callSilent('vendor:publish', ['--tag' => 'cashier-update']);
+        sleep(2);
+        //Enable lock on Cashier::run()
 
+
+        $this->comment('Lock Cashier::run()');
+        $this->lock(CashierRun::class);
```

Comment:
As I understand the Symfony lock documentation, locks are managed in a centralized store. This would mean that inside CashierRun we would need to introduce a locking behaviour too for this to work.

Reply:
I remove this